app/controllers/admin/dashboard_controller.rb

```diff
@@ -16,7 +16,7 @@ def index
       @deletions_enabled     = Setting.open_deletion
       @invites_enabled       = Setting.min_invite_role == 'user'
       @search_enabled        = Chewy.enabled?
-      @version               = Mastodon::Version.to_s
+      @version               = Florence::Version.to_s
```

Comment:
I think informing admins of what version of Mastodon they're using in the dashboard would be useful.
```suggestion
      @version               = Florence::Version.to_s
      @masto_version         = Mastodon::Version.to_s
```